images/win/scripts/ImageHelpers/VisualStudioHelpers.ps1

```diff
@@ -0,0 +1,76 @@
+Function Install-VisualStudio
+{
+    <#
+    .SYNOPSIS
+        A helper function to install Visual Studio.
+
+    .DESCRIPTION
+        Prepare system environment, and install Visual Studio bootstrapper with selected workloads.
+
+    .PARAMETER BootstrapperUrl
+        The URL from which the bootstrapper will be downloaded. Required parameter.
+
+    .PARAMETER WorkLoads
+        The string that contain workloads that will be passed to the installer.
+    #>
+
+    Param
+    (
+        [Parameter(Mandatory)]
+        [String] $BootstrapperUrl,
```

Comment:
Nitpick: perhaps will looks more readable as this:
 [Parameter(Mandatory)] [String] $BootstrapperUrl,
 [Parameter()] [String] $WorkLoads

Reply:
We are using this approach for every other function, so let's leave it as it is if you don't mind.

Reply:
ok